Components/Button.ux

```diff
@@ -0,0 +1,71 @@
+<Panel ux:Class="Material.Button" Alignment="Center" Enabled="true" Raised="true" Height="48" MinWidth="88" Padding="8,0" HitTestMode="LocalBoundsAndChildren" BackgroundColor="White" InkColor="#ccc">
+
+	<string ux:Property="Text" />
+	<bool ux:Property="Enabled" />
+	<bool ux:Property="Raised" />
+	<float4 ux:Property="BackgroundColor"/>
+	<float4 ux:Property="InkColor"/>
+
+	<Rectangle ux:Name="RaisedButton_body" ZOffset="1" Height="36" Padding="8,0" CornerRadius="2">
```

Comment:
@igorls I tested and the `ZOffset`s do not seem to be necessary at all. You only need to put the nodes in the right order, just like layers in photoshop - the ones on top in UX are rendered on top of others.